dolibarr: 16.0.5 -> 17.0.3

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://github.com/Dolibarr/dolibarr/releases.


meta.description for dolibarr is: A enterprise resource planning (ERP) and customer relationship manager (CRM) server

meta.homepage for dolibarr is: https://dolibarr.org/




###### Updates performed
- Version update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/Dolibarr/dolibarr/releases/tag/17.0.0)

- [Compare changes on GitHub](https://github.com/Dolibarr/dolibarr/compare/16.0.4...17.0.0)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- Warning: a test defined in `passthru.tests` did not pass
- found 17.0.0 with grep in /nix/store/1afzd5975xirvi5320sgg7xkh4a9aah1-dolibarr-17.0.0
- found 17.0.0 in filename of file in /nix/store/1afzd5975xirvi5320sgg7xkh4a9aah1-dolibarr-17.0.0

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
1 total rebuild path(s)

1 package rebuild(s)

First fifty rebuilds by attrpath
dolibarr
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/1afzd5975xirvi5320sgg7xkh4a9aah1-dolibarr-17.0.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A dolibarr https://github.com/r-ryantm/nixpkgs/archive/c528d56748146d2c8806deb4eb05929a310e4a2a.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/1afzd5975xirvi5320sgg7xkh4a9aah1-dolibarr-17.0.0
ls -la /nix/store/1afzd5975xirvi5320sgg7xkh4a9aah1-dolibarr-17.0.0/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>dolibarr</li>
  </ul>
</details>

---

###### Maintainer pings

cc @RaitoBezarius for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

RaitoBezarius commented:
Installation mechanism seems to have changed or the test harness is broken for now.

r-ryantm commented:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/project/dolibarr/versions.


meta.description for dolibarr is: A enterprise resource planning (ERP) and customer relationship manager (CRM) server

meta.homepage for dolibarr is: https://dolibarr.org/




###### Updates performed
- Version update

###### To inspect upstream changes



- [Compare changes on GitHub](https://github.com/Dolibarr/dolibarr/compare/16.0.4...17.0.1)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- Warning: a test defined in `passthru.tests` did not pass
- found 17.0.1 with grep in /nix/store/l0bf1qwa9n21mkdk9i17aihq680q862x-dolibarr-17.0.1
- found 17.0.1 in filename of file in /nix/store/l0bf1qwa9n21mkdk9i17aihq680q862x-dolibarr-17.0.1

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
1 total rebuild path(s)

1 package rebuild(s)

First fifty rebuilds by attrpath
dolibarr
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/l0bf1qwa9n21mkdk9i17aihq680q862x-dolibarr-17.0.1 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A dolibarr https://github.com/r-ryantm/nixpkgs/archive/3aefe56dcfe0926fd7e871109ec659fad8109aae.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/l0bf1qwa9n21mkdk9i17aihq680q862x-dolibarr-17.0.1
ls -la /nix/store/l0bf1qwa9n21mkdk9i17aihq680q862x-dolibarr-17.0.1/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>dolibarr</li>
  </ul>
</details>

---

###### Maintainer pings

cc @RaitoBezarius for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

r-ryantm commented:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/project/dolibarr/versions.


meta.description for dolibarr is: A enterprise resource planning (ERP) and customer relationship manager (CRM) server

meta.homepage for dolibarr is: https://dolibarr.org/




###### Updates performed
- Version update

###### To inspect upstream changes



- [Compare changes on GitHub](https://github.com/Dolibarr/dolibarr/compare/16.0.4...17.0.2)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- Warning: a test defined in `passthru.tests` did not pass
- found 17.0.2 with grep in /nix/store/vhbpsz3c8np198k5ffklryzis6mp0mcy-dolibarr-17.0.2
- found 17.0.2 in filename of file in /nix/store/vhbpsz3c8np198k5ffklryzis6mp0mcy-dolibarr-17.0.2

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
1 total rebuild path(s)

1 package rebuild(s)

First fifty rebuilds by attrpath
dolibarr
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/vhbpsz3c8np198k5ffklryzis6mp0mcy-dolibarr-17.0.2 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A dolibarr https://github.com/r-ryantm/nixpkgs/archive/dab70d9f33f2fcc78c99adce2b242709f50bc632.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/vhbpsz3c8np198k5ffklryzis6mp0mcy-dolibarr-17.0.2
ls -la /nix/store/vhbpsz3c8np198k5ffklryzis6mp0mcy-dolibarr-17.0.2/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>dolibarr</li>
  </ul>
</details>

---

###### Maintainer pings

cc @RaitoBezarius for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

r-ryantm committed:
dolibarr: 16.0.5 -> 17.0.3

r-ryantm commented:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://github.com/Dolibarr/dolibarr/releases.


meta.description for dolibarr is: A enterprise resource planning (ERP) and customer relationship manager (CRM) server

meta.homepage for dolibarr is: https://dolibarr.org/




###### Updates performed
- Version update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/Dolibarr/dolibarr/releases/tag/17.0.3)

- [Compare changes on GitHub](https://github.com/Dolibarr/dolibarr/compare/16.0.5...17.0.3)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- Warning: a test defined in `passthru.tests` did not pass
- found 17.0.3 with grep in /nix/store/lymbaaqpcmf52vdl1nxgff6lzxhvz9gm-dolibarr-17.0.3
- found 17.0.3 in filename of file in /nix/store/lymbaaqpcmf52vdl1nxgff6lzxhvz9gm-dolibarr-17.0.3

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
1 total rebuild path(s)

1 package rebuild(s)

First fifty rebuilds by attrpath
dolibarr
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/lymbaaqpcmf52vdl1nxgff6lzxhvz9gm-dolibarr-17.0.3 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A dolibarr https://github.com/r-ryantm/nixpkgs/archive/ec0c7f2799dd2afc2e0a3e03acd6d3bbc2ac3b44.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/lymbaaqpcmf52vdl1nxgff6lzxhvz9gm-dolibarr-17.0.3
ls -la /nix/store/lymbaaqpcmf52vdl1nxgff6lzxhvz9gm-dolibarr-17.0.3/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>dolibarr</li>
  </ul>
</details>

---

###### Maintainer pings

cc @RaitoBezarius for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

RaitoBezarius committed:
nixos/tests/dolibarr: use -X GET instead -X POST to test for redirection

Previously, we were POST-ing the homepage to test for redirection, this is wrong.

We are supposed to GET-ing it.

This is fixed.
